src/System.Net.NameResolution/tests/UnitTests/System.Net.NameResolution.Unit.Tests.csproj

```diff
@@ -5,10 +5,9 @@
     <ProjectGuid>{239347DB-D566-48C9-9551-28AB3AD12EC3}</ProjectGuid>
     <StringResourcesPath>../../src/Resources/Strings.resx</StringResourcesPath>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <NoWarn>0436</NoWarn> <!-- Avoid warnings about type conflicts for types we're building in -->
+    <NoWarn>0436</NoWarn>
```

Comment:
I can fix that up so that future updates don't break it.  This is auto-formatting by MSBuild document model.